app/views/workflows/_edit_form.erb

```diff
@@ -4,7 +4,7 @@
     <%= f.alert_message "Please fix the errors below." %>
 
     <%= f.text_field :name %>
-    <%= f.select :batch_host, OODClusters.map { |key, val| [ "#{val.title} (#{val.resource_mgr_server.host})", key ] }, { selected: @workflow.batch_host, label: "Batch Server" }, { class: "selectpicker" } %>
+    <%= f.select :batch_host, OODClusters.map { |cluster| [ "#{cluster.metadata_config[:title]} (#{cluster.job_config[:host]})", cluster.id ] }, { selected: @workflow.batch_host, label: "Batch Server" }, { class: "selectpicker" } %>
```

Comment:
It might be better to use:

```ruby
cluster.metadata.title
```

instead of

```ruby
cluster.metadata_config[:title]
```

as it allows us in the future to change the underlying config, but provide a metadata getter that can be backwards compatible.